### PR TITLE
Completed dom-creation exercise

### PR DESCRIPTION
**Pull request by rachel-lyy**
Screen Shot 2022-03-17 at 8 50 55 PM

### CODE REVIEW

File: dom-creation/main.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  {
+    number: '009',
+    name: 'Blastoise',
+    description: 'It crushes its foe under its heavy body to cause fainting. In a pinch, it will withdraw inside its shell.',
+    imageUrl: 'images/blastoise.png'
+  }
+];
+
+function renderPokemon(pokemon) {
+  for (let i = 0; i < pokemon.length; i++) {
```
Comment (BrettAlbright) [inline dom-creation/main.js]: the loop should be outside the function and call the function with one object at a time

File: dom-creation/main.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    var paragraph = document.createElement('p');
+    paragraph.textContent = pokemon[i].description;
+
+    column.appendChild(card);
+    card.appendChild(pokemonImg);
+    card.appendChild(textDiv);
+    textDiv.appendChild(nameH2);
+    textDiv.appendChild(numH3);
+    textDiv.appendChild(paragraph);
+    document.querySelector('.row').appendChild(column);
```
Comment (BrettAlbright) [inline dom-creation/main.js]: you don't want to append in the function, you want to return the DOM tree and append it outside of the function.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Great work on the functionality! A couple updates are needed.

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**